src/vm/threads.h

```diff
@@ -3740,13 +3739,15 @@ class Thread: public IUnknown
     // to identifying redirection targets
     static BOOL IsAddrOfRedirectFunc(void * pFuncAddr);
 
+#endif //defined(FEATURE_HIJACK) && defined(PLATFORM_UNIX)
```

Comment:
And also, the #endif should not be moved from its original location - because of it matched how the implementation is ifdefed out.